### PR TITLE
Dependency updates + eject CRA

### PR DESCRIPTION
**Pull request by quietbits**
Major updates:
- Require Node v16 (from Node v14)
- React v18
- Eject CRA - `react-scripts` v5 uses webpack 5 which doesn't include node.js modules by default anymore. We had to eject to update webpack config manually to include them.
- Latest SDS with some updates
- Added `trezorStellarPlugin` locally, it's no longer available in `extended` package we used before

### CODE REVIEW

File: config/env.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+"use strict";
```
Comment (piyalbasu) [inline config/env.js:1]: assuming all this `config/*` stuff is from CRA eject

Comment (piyalbasu) [inline config/env.js:1]: assuming all this `config/*` stuff is from CRA eject

Comment (quietbits) [reply config/env.js:1]: Yup! It's quite a lot and is probably way too complex for what we need, but it works. :woman_shrugging:

Comment (piyalbasu) [reply config/env.js:1]: Wow. This is more scaffolding than I thought there would be. How do you feel about this? Do you think it's too confusing having all this "extra" stuff in the repo?

Comment (piyalbasu) [reply config/env.js:1]: Wow. This is more scaffolding than I thought there would be. How do you feel about this? Do you think it's too confusing having all this "extra" stuff in the repo?

Comment (quietbits) [reply config/env.js:1]: I agree that it is way too much for our projects. Not sure if it's worth spending time to add a custom webpack now or do some thinking about our future tools. There are other options available that might be better?

**Comment by quietbits** [comment]
Closing in favor of https://github.com/stellar/account-viewer-v2/pull/387

**Pull request closed without merging**